src/views/shared/TextBox.js

```diff
@@ -22,8 +22,24 @@ define([
 function (Okta, BrowserFeatures) {
 
   var { TextBox } = Okta.internal.views.forms.inputs;
+  var { tpl } = Okta;
 
   return TextBox.extend({
+    template: tpl(
```

Comment:
Why are we adding this template ?

Reply:
Overwrite template in courgea's TextBox to add `aria-hidden` attribute.